hyperspy/_signals/signal1d.py

```diff
@@ -1532,6 +1533,63 @@ def estimating_function(spectrum,
 
     estimate_peak_width.__doc__ %= (SHOW_PROGRESSBAR_ARG, NUM_WORKERS_ARG)
 
+    def get_intensity(self,
+                           indexes,
+                           start=None,
+                           stop=1.0,
+                           factor=1.0,
+                           norm=None,
+                           minimum_intensity=None,
+                           **kwargs):
+        """
+        A generic function for getting the intensity of a line at different positions in the Signal1D.
+
+        This function is used mostly for plotting markers and the output can be directly
+        passed to a `MarkerCollection` object using the `MarkerCollection.from_signal()` method.
+
+        Parameters
+        ----------
+        indexes : list of float
```

Comment:
Rename to `values` to follow a similar syntax as the hyperspy axis (`indices` versus `values`)